### PR TITLE
feat: add sitemap.xml tnd robots.txt to react

### PR DESCRIPTION
**Pull request by timotheeg**
## Problem
The new formSG version broke the links
* [https://form.gov.sg/robots.txt](https://form.gov.sg/robots.txt)
* [https://form.gov.sg/sitemap.xml](https://form.gov.sg/sitemap.xml)

## Solution
Restore the assets

### CODE REVIEW

File: frontend/public/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:image="http://www.google.com/schemas/sitemap-image/1.1">
+  <url>
+    <loc>https://form.gov.sg/</loc>
+    <priority>1.0</priority>
+    <image:image>
+       <image:loc>
+         https://form.gov.sg/static/images/landing/restricted__build-demo.png
```
Comment (timotheeg) [inline frontend/public/sitemap.xml:9]: I carried that image from angular, but actually the screenshot is wrong, we'll need to take another @staceytan1998 🤔

**Review by karrui — APPROVED** [review]
lgtm.

**Comment by karrui** [comment]
also also, no other work needed for react yea? since express knows the entirety of `dist/frontend` are static assets

**Comment by timotheeg** [comment]
> also also, no other work needed for react yea? since express knows the entirety of dist/frontend are static assets

Yep, that's right! 😄

**Comment by staceytan1998** [comment]
Sitemap image here: https://www.figma.com/file/9oE4ECjdeZQ1RBpwZVtnB3/Form-Design-Master?node-id=13798%3A136037 

Is this sufficient or do I need to export it in some format?

**Comment by timotheeg** [comment]
Thanks @staceytan1998 . I have exported the asset. I'll merge this PR when the build passes.

**Pull request merged**